### DISCUSSION TITLE
G__~1.EXE: warning: /bigobj: linker input file unused because linking not done G__~1.EXE: error: /bigobj: linker input file not found: No such file or directory ninja: build stopped: subcommand failed.

### DISCUSSION THREAD

**Discussion by dEN5-tech** [slint-ui/slint#4536] [Q&A] [answered]
Hello.

![clion64_4BRMfAE67x](https://github.com/slint-ui/slint/assets/54153548/bf190557-04b1-4c46-a88a-e8ade7fcfe73)

How fix this?

**Comment by ogoffart** [comment]
/bigobj is a MSVC option.
It is set there:
https://github.com/slint-ui/slint/blob/ddeb8243564378c02c530d54b3e6e37314b841e9/api/cpp/CMakeLists.txt#L151-L152

But from the error, it looks like you're using mingw.  How can the option be set since we have a `if (MSVC)`

How did you configure the project? Why do cmake believe you are using msvc but then runs mingw?

**Reply by dEN5-tech** [reply]
### 
![image](https://github.com/slint-ui/slint/assets/54153548/9428d836-e808-44b9-8c4f-62a2e43cfe96)

i use default

**Reply by dEN5-tech** [reply, marked as answer]
i select vs 2019, and get message of qt6code.dll,gui and etc not found
